spec/fixtures/config/ood_portal/portal.yml

```diff
@@ -3,6 +3,6 @@
 # Portal configuration
 #
 
-listen_addr_port: 8080
+listen_addr_port: 8083
```

Comment:
These 2 port changes will break E2E tests because the Docker port binding is for 8080 so that browser tests run on the host can access port 8080 of the container.  Is there a particular reason changed this and `port`?

Reply:
I introduced that so that we can confirm the output log files.  I'll revert it once we know this works.